bitrise.yml

```diff
@@ -59,7 +59,7 @@ workflows:
             # Check for unhandled errors
             go get -u -v github.com/kisielk/errcheck
             # Go lint
-            go get -u -v github.com/golang/lint/golint
```

Comment:
pls wire in our usual go test steps

Reply:
Fixed in:
https://github.com/bitrise-io/go-steputils/pull/12/commits/c3b411c34e53d03c8d48d95839aa90ddb3901635